python/ray/data/read_api.py

```diff
@@ -1490,6 +1490,9 @@ def read_tfrecords(
     tf_schema: Optional["schema_pb2.Schema"] = None,
     shuffle: Union[Literal["files"], None] = None,
     file_extensions: Optional[List[str]] = None,
+    batch_size: Optional[int] = None,
```

Comment:
sure, we are kind of leaking to the users that there's an underlying fast and slow im,plementation, which might be confusing to the user

Reply:
good point. i think since we will mention that this is used for only the fast path in the docstring anyway, i was thinking it would be good to surface that fact explicitly with the parameter name as well.